example/ML_test/RANS-uq_int-opt.in

```diff
@@ -0,0 +1,179 @@
+                                     -1.00000000
+                                     -1.00000000
+                                     -1.00000000
+                                     -1.00000000
+                                     -1.00000000
+                                     -1.00000000
+                                     -1.00000000
+                                     -1.00000000
+                                     -1.00000000
+                                     1.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+                                     0.00000000
+
+responses
+    id_responses = 'OPTIM_R'
+    objective_functions = 1
+    nonlinear_inequality_constraints = 1  # Exp val of M, T
+  	nonlinear_inequality_lower_bounds = 23000.
+  	nonlinear_inequality_upper_bounds = 1.e+50
+    numerical_gradients
+        method_source dakota
+        interval_type forward
+        fd_gradient_step_size = 1.e-3
+    no_hessians
+
+interface
+    id_interface = 'OPTIM_I'
+    analysis_driver = 'bash driver_uq_int.sh'
+        fork # asynchronous evaluation_concurrency = 4
+        work_directory named 'run_uq_int'
+        link_files 'runModel.py' 'general_gradients.cfg' 'make_responses' 'postproc'
+	copy_files '../../ML_code/train.py' '../../ML_code/train.dat' '../../ML_code/shaowuML.py' '/../../ML_code/shaowuML.pyc' '/../../ML_code/config.pkl' '/../../ML_code/ensemble_model.pkl' '/../../ML_code/normalize_dict.pkl'
```

Comment:
It appears the pickle files are overwritten when train.py is called by the analysis driver script. Do we need to copy them?

Reply:
There is another alternative where we can link the files but the output file from ML subroutine beta_for_SU2.dat will be generated in the MULTIF/ML_code. However, this would mean that asynchronous concurrency feature of Dakota cannot be used.